runtime/browser/demo/recipes.manifest

```diff
@@ -26,12 +26,24 @@ recipe
 
 # Create shortlist with [product, ...] and suggest similar products from [person]'s wish list
 recipe
-  Chooser.choices -> Recommend.recommendations
-  Chooser.resultList -> ShowProducts.list
-  Chooser.resultList -> Recommend.known
+  # TODO: replace with this once connection constraints are fixed.
```

Comment:
Added a hack to make this work again for now. It uses a manifest instead of the old particle loader.